crypto/x509/x509_vfy.c

```diff
@@ -2545,6 +2549,298 @@ X509_STORE_CTX_lookup_crls_fn X509_STORE_CTX_get_lookup_crls(X509_STORE_CTX *ctx
     return ctx->lookup_crls;
 }
 
+/* Begin special code for automatically fetching CRLs */
+/* 
+ * This is copied from `apps.c`.
```

Comment:
See: https://github.com/mirrorfitness/openssl-LLS-fork/blob/main-lululemon-studio/apps/apps.c

You can find the original function names by dropping the `lulu_studio_copies__` prefix.